buildconfig/stubs/pygame/event.pyi

```diff
@@ -17,10 +17,9 @@ class Event:
     dict: Dict[str, Any]
     __dict__: Dict[str, Any]
     __hash__: None  # type: ignore
-    @overload
-    def __init__(self, type: int, dict: Dict[str, Any]) -> None: ...
-    @overload
-    def __init__(self, type: int, **attributes: Any) -> None: ...
+    def __init__(
+        self, type: int, dict: Dict[str, Any] = ..., **kwargs: Any
```

Comment:
What does having some default to ellipses mean in type annotation?

Reply:
It indicates that the argument is a "default argument" but does not specify the default type